src/flow-control/log-timings.js

```diff
@@ -0,0 +1,57 @@
+const Rx = require('rxjs');
+const _ = require('lodash');
+const { bufferCount, take } = require('rxjs/operators');
+const BaseHandler = require('./base-handler');
+
+module.exports = class LogTimings extends BaseHandler {
+    printExitInfoTimingTable(exitInfos) {
+        const exitInfoTable = _(exitInfos)
+            .map(({ command, timings, index, killed, exitCode }) => {
+                const readableDurationMs = (timings.endDate - timings.startDate).toLocaleString();
+                return {
+                    'call-index': index,
+                    name: command.name,
+                    duration: `~${readableDurationMs}ms`,
```

Comment:
Why the `~`? Is there any rounding somewhere I'm not spotting?

Reply:
its just because the duration returned from in the exit info is more accurate, down to the nanosecond, and the two durations might not match but I guess adding this tilde just adds noise to the logs, I'll remove it